torch/csrc/jit/pickler.cpp

```diff
@@ -408,6 +423,7 @@ void Pickler::pushLong(const std::string& data) {
         data.size() > std::numeric_limits<uint32_t>::max(),
         "Cannot pickle a long with a size larger than 4 bytes")
     push<PickleOpCode>(PickleOpCode::LONG4);
+    // TODO: should this be uint32_t instead?
```

Comment:
Yeah... this looks wrong.  See https://github.com/python/cpython/blob/2.7/Lib/pickletools.py#L677

Reply:
Thanks. I took this TODO out of the code, and uploading a shortish diff to deal with this.